### PR TITLE
catch ImproperlyConfigured exc in user_login view

### PR DESCRIPTION
**Pull request by pacahon**

### CODE REVIEW

**Comment by pacahon** [comment]
Not sure it's really necessary but we could save some LOC if use one try-catch block both for checking permissions and login action.

**Comment by pacahon** [comment]
Btw I've fixed this line accidentally https://github.com/skorokithakis/django-loginas/blob/cf5dccc69acf1d7bfc383cc2eda088f779e9ed21/loginas/utils.py#L60 Let me know to revert it if I'm wrong. I'm not good at English :)

**Comment by skorokithakis** [comment]
This looks good to me, apart from the minor grammar error. If you've tested this in the admin and it works properly, I'm good to merge!

File: loginas/tests/tests.py [hunk 1]
```diff
@@ -66,6 +68,16 @@ def can_login_as_always_raise_permission_denied(request, user):
     raise PermissionDenied("You can't login as target user")
 
 
+class WrongAuthBackend:
+    """
+    An authentication backend is a class that implements two required methods: get_user(user_id) and
+    authenticate(**credentials). Unfortunately, some libraries don't comply with this interface (e.g.
+    `django-rules` with ObjectPermissionBackend) and omit required `get_user` method.
```
[the remark this answers is not in the document]
Comment (pacahon) [reply loginas/tests/tests.py]: fixed

**Comment by skorokithakis** [comment]
Merged, thanks!

**Comment by pacahon** [comment]
I've tested it on my own project with custom auth system, works fine. Also I can test it manually on bare django project with default configuration if the tests are not enough.

**Comment by skorokithakis** [comment]
If you get a chance, please do, as it would be good to know it doesn't break anything in the default configuration, but it's not urgent. Thanks for the help!

**Pull request merged**